### DISCUSSION TITLE
Custom functions on BigQuery?

### DISCUSSION THREAD

**Discussion by hector** [hasura/graphql-engine#7524] [Question] [answered]
Is it possible to call custom function in BigQuery, the same way it is possible in PostgreSQL?
https://hasura.io/docs/latest/graphql/core/databases/postgres/schema/custom-functions.html

Are custom functions supported by Hasura in other database right now?

**Comment by BenoitRanque** [comment, marked as answer]
From the documentation: https://hasura.io/docs/latest/graphql/core/databases/bigquery/index.html#supported-features
> Hasura currently only supports queries on BigQuery.

Other functionality is in the works, but the short answer is that no, as of this moment custom functions are not supported on BigQuery.

**Comment by dariovillalon** [comment]
I have the same issue, I need to filter the data by some dates and I'm making some calculations with them that need to be done with the filters applied, do you have a work around for this?

Because I can't imagine how to solve this right now
